templates/core/Resources/_ti/extensions/String.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2018 by Axway, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+
+'use strict';
+
+// Determines if given object is not of type function.
+function isNotFunction(object) {
```

Comment:
I'm not sure a function dedicated to determining if something is a function is beneficial. It's such a simple statement.

From a performance perspective, it's fine in this case, but I found that if you were to do something like this on a greater scale, it would be come an extremely hot function and thus should be avoided.